pandas/core/resample.py

```diff
@@ -1606,6 +1634,28 @@ def _get_range_edges(first, last, offset, closed='left', base=0):
     return first, last
 
 
+def _get_period_range_edges(first, last, offset, closed='left', base=0):
+    """
+    Adjust the provided Period range edge values to the appropriate edge
+    values for the given offset parameters.
```

Comment:
can you give a 1 sentence on what this is doing. imagine you have never seen this and explain in simplest possible terms.

Reply:
updated, please take a look and provide any feedback. the timestamp one is a couple sentences, but I think it's clear.